src/test/java/POM/pages/BasePage.java

```diff
@@ -42,6 +47,10 @@ public AllSongsPage clickOnAllSongs(){
         driver.findElement(allSongsMenuItemLocator).click();
         return new AllSongsPage(driver);
     }
+    public MyAlbumsPage clickOnAlbums(){
+        driver.findElement(albumsMenuLocator).click();
+        return new MyAlbumsPage(driver);
+    }
 
 
-}
+}
```

Comment:
`Base Page \ Base Class` — Implements the necessary methods for work with webdriver.

We will store the web elements present on each page in the corresponding java class.

The methods `getUserAvatar ()`,`isUserAvatarDisplayed() `must be stored in the `class HomePage` because the avatar is visible on the web site -home page

The methods `isSongPlaying()`,`clickOnAllSongs()` and `clickOnAlbums()` must be stored in the  `class AllSongsPage` because the song is visible on the web site -songs page


For example

![image](https://user-images.githubusercontent.com/104761482/203916633-798a29c9-6ca7-49b8-9dbc-083aca108108.png)